controllers/ndbserver_controller_helpers.go

```diff
@@ -16,18 +16,29 @@ import (
 func getNDBServerDatabasesInfo(ctx context.Context, ndbClient *ndb_client.NDBClient) (databases []ndbv1alpha1.NDBServerDatabaseInfo, err error) {
 	log := log.FromContext(ctx)
 	log.Info("Fetching and converting databases from NDB")
-
+	// Difference between the vars 'dbs' and 'databases'
```

Comment:
could we instead use something more intuitive like: db_responses, db_objects or something like that